### ISSUE TITLE
No function clause for {:caught_up, reference}

### ISSUE THREAD

**Issue by pauldemarco** [NFIBrokerage/volley#8]
Doesn't seem to work with some of the later versions of EventStoreDB:
```
21:20:39.809 [error] GenServer :my_producer terminating
** (FunctionClauseError) no function clause matching in Volley.InOrderSubscription.handle_info/2
    (volley 1.0.0) lib/volley/in_order_subscription.ex:415: Volley.InOrderSubscription.handle_info({:caught_up, #Reference<0.1012077486.440139777.47607>}, %{position: %Spear.Event{id: "b54a0812-4813-4e84-ae69-dacc02ea47fc", type: "hello", body: %{"foo" => "bar"}, link: nil, metadata: %{stream_name: "my_stream", stream_revision: 0...
```

**Comment by the-mikedavis** [comment]
Yep this is from the somewhat new caught-up and fell-behind notifications in eventstore 23.10 and above. Fixed in https://github.com/NFIBrokerage/volley/pull/9

**Comment by the-mikedavis** [comment]
Released in v1.0.1
